phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/RepositoriesPanel.java

```diff
@@ -130,19 +132,20 @@ private void populateRepositoryGrid() {
             dialog.setContentText(tr("Please add the new repository:"));
 
             Optional<String> result = dialog.showAndWait();
-            result.ifPresent(newRepository -> {
-                repositories.add(0, newRepository);
-
-                this.save();
-
-                repositoryManager.addRepositories(0, newRepository);
-            });
+            //            result.ifPresent(newRepository -> {
```

Comment:
This is only commented out because I want to replace this code by a new GUI in one of my next PRs (#776)